README.md

```diff
@@ -245,6 +245,10 @@ messages are emitted via webpack which is not affected by this flag.
 You can squelch certain TypeScript errors by specifying an array of diagnostic
 codes to ignore.
 
+#### reportFiles *(string[]) (default=[])*
+
+Only report errors on files matching these glob patterns.
```

Comment:
Could you add an example here please?  Examples rule the world when it comes to docs in my experience.

Reply:
I have added an example